### PR TITLE
feat(pagination): implement InstantSearch.css

### PR DESCRIPTION
**Pull request by francoischalifour**
## Specs

https://instantsearch-css.netlify.com/widgets/pagination/

## Stories

https://deploy-preview-3119--algolia-instantsearch.netlify.com/v2/dev-novel/?selectedStory=Pagination.default

### CODE REVIEW

File: src/widgets/pagination/pagination.js [hunk 1]
```diff
@@ -146,9 +159,11 @@ export default function pagination({
   labels: userLabels = defaultLabels,
   cssClasses: userCssClasses = {},
   maxPages,
-  padding,
-  showFirstLast = true,
-  autoHideContainer = true,
+  padding = 3,
```
Comment (francoischalifour) [inline src/widgets/pagination/pagination.js]: The default value should be removed (see #2866).

File: src/components/Pagination/__tests__/Pagination-test.js [hunk 2]
```diff
@@ -13,14 +13,17 @@ describe('Pagination', () => {
...
-      next: 'next',
-      first: 'first',
-      last: 'last',
-      active: 'active',
-      disabled: 'disabled',
+      firstPageItem: 'firstPageItem',
+      lastPageItem: 'lastPageItem',
+      previousPageItem: 'previousPageItem',
+      nextPageItem: 'nextPageItem',
+      pageItem: 'pageItem',
```
Comment (bobylito) [inline src/components/Pagination/__tests__/Pagination-test.js]: Why not `item` instead of `pageItem`? It would be more consistent.

Comment (francoischalifour) [reply src/components/Pagination/__tests__/Pagination-test.js]: I based this name on the CSS class `ais-Pagination-item--page` → `pageItem`.

Another example of a modifier would be `ais-Pagination-item--firstPage` → `firstPageItem`.


Should we just go for `item` in this specific case, then?

**Comment by francoischalifour** [comment]
Done @bobylito.

**Comment by bobylito** [comment]
I'll let you merge the changes :)

**Pull request merged**